README.md

```diff
@@ -10,7 +9,7 @@ js-multihash
 ### In Node.js through npm
 
 ```bash
-$ npm install --save multihashes # node the name of the module is multihashes
+$ npm install --save multihashes
```

Comment:
unfortunately `multihash` exists on npm, so there is space for confusion. Also, this should not be the focus of this PR